lib/scripts/get_tags.py

```diff
@@ -56,14 +59,14 @@
 user_tags = os.environ.get('DD_TAGS', None)
 if user_tags:
     try:
+        user_tags = user_tags.replace(" ", "_")
         user_tags = user_tags.split(',')
         for tag in user_tags:
             tags.append(tag)
     except Exception as e:
         print("there was an issue parsing the tags in DD_TAGS: {}".format(e))
 
```

Comment:
```suggestion
tags = [ tag.replace("", " _") for tag in tags ]
```
Let's do it once here instead.
I didn't test the following code but it should be close.